data/sql/updates/pending_db_world/rev_1686872651542486500.sql

```diff
@@ -0,0 +1,5 @@
+--
+DELETE FROM `smart_scripts` WHERE `entryorguid`=173 AND `source_type`=2;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(173, 2, 0, 1, 46, 0, 100, 0, 0, 0, 0, 0, 0, 12, 2056, 3, 300000, 0, 0, 0, 8, 0, 0, 0, 0, 1077, 1539, 28.89, 0, 'Areatrigger - On Trigger - Summon Creature \'Ravenclaw Apparition\' (The Dead Fields)'),
+(173, 2, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 15, 437, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Areatrigger - On Link - Call Area explored or event happens');
```

Comment:
Use INVOKER_PARTY as the target, otherwise each player in the party will need to enter the area, but since the Summon has a cooldown it can only happen once every 5mins

Reply:
> Use INVOKER_PARTY as the target, otherwise each player in the party will need to enter the area, but since the Summon has a cooldown it can only happen once every 5mins

Perfect, when I get home, I try to correct it. If it lets me from here I'll do it, but I'm not sure. I will try.